apps/meteor/app/api/server/v1/misc.ts

```diff
@@ -392,7 +392,7 @@ API.v1.addRoute(
 API.v1.addRoute(
 	'pw.getPolicy',
 	{
-		authRequired: true,
+		authRequired: false,
```

Comment:
I don't think that it's a problem, but check with Backend if it's ok to not require auth, please

Reply:
Checked with @matheusbsilva137 and @xxxmadxxx. They approved it.